### DISCUSSION TITLE
Where do I need to put my custom source, platform, etc. banners so that this theme will use them?

### DISCUSSION THREAD

**Discussion by Dadditude** [MCC321-QC/eMixedNiteMC#30] [Q&A] [answered]
Where do I need to put my custom source, platform, etc. banners so that this theme will use them?

I've got them in the appropriate subdirectory of the ThemeExtras user banners directory, but they aren't being used. Any help would be greatly appreciated.

**Comment by MCC321-QC** [comment, marked as answer]
Sorry for being almost a year late, banners are implemented directly in the theme and not with ThemeExtras plugin so banners go in the theme installation directory. 

the source banners goes here : ..\Playnite\Themes\Desktop\eMixedNiteMC\Images\Banners\Stores
and the platforms banners here : ..\Playnite\Themes\Desktop\eMixedNiteMC\Images\Banners\PlatformsExtra

You can easily open the theme installation folder from your installed themes in your Add-ons menu!
